webdriver-spec.html

```diff
@@ -7996,7 +8013,7 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+ <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
```

Comment:
No, this needs to be the top-level browsing context since alerts are modal and tied to the browser chrome window.